### DISCUSSION TITLE
板块识别

### DISCUSSION THREAD

**Discussion by whwususu** [PaddlePaddle/PaddleX#2344] [Q&A] [answered]
我们用官方的文档提供的方法，自己新建了几个label,以[RT-DETR-H_layout_17cls.yaml] 学习模型为基础，训练出了新模型，但是新模型里面label只有新添加的，并没有继承原来模型的label，这个意思不就说明了我们不能在原模型的基础上进行训练么。我看了所有文档，没有找到原因，麻烦大家帮忙解答下，是我理解错误了，还是设置错误了，导致无法继承原来的模型，同时加入我们需要新训练的，谢谢！

**Comment by cuicheng01** [comment, marked as answer]
新模型会遗忘过去的模型的标签，只能拟合新的数据的情况。
